src/main/java/stsjorbsmod/actions/PermanentlyModifyDamageAction.java

```diff
@@ -28,12 +28,6 @@ public void update() {
                 c.isDamageModified = false;
             }
         }
-
-        for(var1 = GetAllInBattleInstances.get(uuid).iterator(); var1.hasNext(); c.baseDamage = c.misc) {
```

Comment:
I don't think the original `ModifyDamageAction` does what we want here, since it doesn't decrement `misc` and so isn't going to cooperate with `RitualDaggerAction`. I'd recommend not extending from it at all, personally; it's just not saving that much code.